### DISCUSSION TITLE
Enable dates displayed in RAC Calender that fall outside the current month

### DISCUSSION THREAD

**Discussion by brandonpittman** [adobe/react-spectrum#4532] [Q&A]
Is there a way to make the dates from previous and next months that show up in a calendar enabled?

There's logic in useCalendarState that makes them disabled, but as a convenience, I'd like to let the user click any available date that's visible in the UI.

**Comment by LFDanLu** [comment]
This is a use case we are tracking here: https://github.com/adobe/react-spectrum/issues/3257.
